docs/Troubleshooting.md

```diff
@@ -11,31 +11,11 @@ Try using the debugging support built into Node.
 
 Place a `debugger;` statement in any of your tests, and then, in your project's directory, run:
 
-```
-node --inspect-brk node_modules/.bin/jest --runInBand [any other arguments here]
+`node inspect node_modules/.bin/jest`
```

Comment:
`node inspect` is not the same as `node --inspect`. I'd keep the parts with `--inspect` and just replace the parts about `node-inspector`